command-line/command-line-advanced.md

```diff
@@ -41,11 +41,17 @@ Teach people some handier tools for advanced file searching and processing
 - [Filenames and Pathnames in Shell: How to do it Correctly](https://www.dwheeler.com/essays/filenames-in-shell.html)
 - [An Awk Primer/Awk Command-Line Examples](https://en.wikibooks.org/wiki/An_Awk_Primer/Awk_Command-Line_Examples)
 - [I/O Redirection](http://wiki.bash-hackers.org/syntax/redirection)
-- [Video](https://www.youtube.com/watch?v=NodKFuUocQY): About 22 minutes. Covers command aliases, whereis, service, passwd, df, du, wget, who,ping,shutdown commands 
+- [Video](https://www.youtube.com/watch?v=NodKFuUocQY): About 22 minutes. Covers command aliases, whereis, service, passwd, df, du, wget, who,ping,shutdown commands
+- [Video](https://www.youtube.com/watch?v=nLa6jAbULe8): About 8 minutes. Covers command cut, &&, sort, uniq, wc, grep commands. 
+- [Video](https://www.youtube.com/watch?v=CdWazJlfue8): About 2 minutes. Covers command pushd, popd commands.
+- [Video](https://www.youtube.com/watch?v=yCTnihfbPCo): About 55 minutes. Covers command grep, sed, awk, tar, less, gzip commands.
```

Comment:
55 minutes!! Do they need to watch the whole thing?

Reply:
Can they skip to certain sections?

Reply:
@alodahl if the audience wants to learn only mention command then they can skip certain parts otherwise they can watch the whole video to learn more commands.

Reply:
Can you please give learners a link to the moment they should start watching, or tell them the minutes:seconds they should jump to to learn these specific ones? I can’t ask them to spend 2 hours on these videos.

Reply:
> Can you please give learners a link to the moment they should start watching, or tell them the minutes:seconds they should jump to to learn these specific ones? I can’t ask them to spend 2 hours on these videos.

@alodahl ma'am what I need to do can u elaborate this?

Reply:
for any video longer than 5 minutes, it would be respectful of the apprentices' time if you could tell them when in the video to find the command you're referencing by minutes and seconds, ie:
3:15 grep
4:08 sed
5:55 - 7:30 - awk, tar, sed, gzip

Reply:
they can decide for themselves if they want to watch the rest

Reply:
ok i will do that

Reply:
Thank you!

Reply:
done

Reply:
@alodahl  I think that the timestamp should have the link to the video to that particular time interval so that user don't have to manually go to that time.

Reply:
> @alodahl I think that the timestamp should have the link to the video to that particular time interval so that user don't have to manually go to that time.

@Kundan28 is there any way to achieve this in markdown.

Reply:
Yes, it's possible.

Reply:
how to achieve that i will do it can u tell me

Reply:
Let's wait for @alodahl to first comment on that...then we will continue with it.

Reply:
ok but I like ur idea to link timestamp with videos

Reply:
that is totally possible... its a sharing option in youtube. I'll choose my own start time if, for example, the narrator spends a long time on introductions before getting to it, and I feel its not helpful for apprentices.  However, because each is a pretty long list, I think the way you did here is perfect.

Reply:
@Kundan28  @alodahl  thanx because I don't know about this feature

Reply:
Okay.